src/main/java/mate/academy/connection/ConnectionUtil.java

```diff
@@ -0,0 +1,27 @@
+package mate.academy.connection;
+
+import java.sql.Connection;
+import java.sql.DriverManager;
+import java.sql.SQLException;
+import java.util.Properties;
+
+public class ConnectionUtil {
+    private static final String DB_URL = "jdbc:mysql://localhost:3306/jdbc";
+    private static final Properties DB_PROPERTIES;
+
+    static {
+        DB_PROPERTIES = new Properties();
+        DB_PROPERTIES.put("user", "root");
+        DB_PROPERTIES.put("password", "*****");

```

Comment:
The password is hardcoded in the source code. It's a good practice to avoid hardcoding sensitive information like passwords. Consider using environment variables or a secure vault to manage sensitive data.